savant/utils/artist/artist_gpumat.py

```diff
@@ -147,6 +147,10 @@ def add_bbox(
             left, top, right, bottom = bbox.visual_box(
                 PaddingDraw(*padding), border_width, self.max_col, self.max_row
             ).as_ltrb_int()
+
+            if (right - left) == 0 or (bottom - top) == 0:
+                raise ValueError('Wrong bbox size.')
```

Comment:
Maybe make the check like so

```suggestion
            if (right - left) < 1 or (bottom - top) < 1:
                raise ValueError('Wrong bbox size.')
```

since we can't rely on `bbox.visual_box()` to return sane values.

Same for the width and height check in `blur()`.

Reply:
ok